csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -511,11 +557,18 @@ Marlin(const int4 *__restrict__ A, // fp16 input matrix of shape mxk
 
   // Register storage for double buffer of shared memory reads.
   FragA frag_a[2][thread_m_blocks];
-  I4 frag_b_quant[2];
+  I4 frag_b_quant[2][b_thread_vecs];
   FragC frag_c[thread_m_blocks][4][2];
   FragS frag_s[2][4];        // No act-order
   FragS act_frag_s[2][4][4]; // For act-order
 
+  // if (blockIdx.x == 0 && threadIdx.x == 0) {
```

Comment:
This looks like it is left over from debugging -- remove it before merging the PR?

Reply:
@alexm-nm

Reply:
Removed